src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/sourcelink.html

```diff
@@ -0,0 +1,7 @@
+{% if show_source and  has_source and sourcename %}
+<div class="tocsection sourcelink">
+    <a href="{{ pathto('_sources/' + sourcename, true)|e }}">
+        <i class="fas fa-code"></i> {{ _("Show Source") }}
```

Comment:
```suggestion
        <i class="fas fa-file-lines"></i> {{ _("Show Source") }}
```

what do you think about this icon instead?

Reply:
I don't see this icon in 5.15.4 but only in 6. where did you find it ? 
https://fontawesome.com/v5/search?q=file-lines&s=solid%2Cbrands

Reply:
how about file-alt? https://fontawesome.com/v5/icons/file-alt?s=solid

Reply:
LGTM